src/libbson/tests/test-bson-vector.c

```diff
@@ -123,6 +123,21 @@ append_vector_packed_bit_from_packed_array (
          ASSERT (bson_iter_next (&copy_iter));
          uint8_t packed_byte = (uint8_t) bson_iter_as_int64 (&copy_iter);
          ASSERT (bson_vector_packed_bit_view_write_packed (view, &packed_byte, 1, i));
+
+         // Read back the packed byte, interpret any masking as a conversion failure.
+         uint8_t packed_byte_check;
+         ASSERT (bson_vector_packed_bit_view_read_packed (view, &packed_byte_check, 1, i));
+         if (packed_byte != packed_byte_check) {
+            bson_set_error (error,
+                            TEST_ERROR_DOMAIN,
+                            TEST_ERROR_CODE,
+                            "byte at index %" PRId64
+                            " with value 0x%02X included write to masked bits (reads as 0x%02X)",
+                            (int64_t) i,
```

Comment:
```suggestion
                            "byte at index %zu with value 0x%02X included write to masked bits (reads as 0x%02X)",
                            i,
```
`%zu` may be used to avoid the cast from `size_t`.